gix-features/Cargo.toml

```diff
@@ -67,6 +66,8 @@ crc32 = ["dep:crc32fast"]
 zlib = ["dep:flate2", "flate2?/rust_backend", "dep:thiserror"]
```

Comment:
Should we be turning on `rust_backend` when using `zlib-rs`?

Reply:
The rust_backend is another pure rust implementation that is much much slower than other backend, it's only there as a fallback

Reply:
Right, but the `zlib-rs` feature pulls in the `zlib` feature, and enables the other Rust backend, which we probably don’t want? I’m wondering if that might somehow be part of the CI problems.